src/oic/oic/provider.py

```diff
@@ -247,9 +247,12 @@ def __init__(self, name, sdb, cdb, authn_broker, userinfo, authz,
         self.extra_claims = extra_claims
         self.extra_scope_dict = extra_scope_dict
 
+        self.register_endpoint = None
         for endp in self.endp:
             if endp.etype == 'registration':
                 endpoint = urljoin(self.baseurl, endp.url)
+                DeprecationWarning("Using `register_endpoint` is deprecated, please use "
```

Comment:
Why only create a `DeprecationWarning` instance? Shouldn't it be used with warning.warn(...) ?

Reply:
Yeah, dumb mistake :)